### PR TITLE
google java style tuning for comments above a public class

### PR DESCRIPTION
**Pull request by kunkun-tang**

### CODE REVIEW

File: azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java [hunk 1]
```diff
@@ -30,12 +30,12 @@
 import org.apache.log4j.Logger;
 
 /**
- * This class is used for creating DB tables by specifying where the create scripts are. The
+ * Creating DB tables by specifying where the create scripts are. The
```
Comment (HappyRay) [inline azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java]: change to "Creates"

How about just "Creates DB tables."?

File: azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java [hunk 2]
```diff
@@ -30,12 +30,12 @@
...
  * input to this class is a folder path, which includes all create table sql scripts. The script's
  * name should follow: create.[table_name].sql in order to be identified. This class is used for
  * unit test only for now.
  *
- * Todo kunkun-tang: We need to fix some reliability issues if we rely on this class to
+ * <p>Todo kunkun-tang: We need to fix some reliability issues if we rely on this class to
```
Comment (HappyRay) [inline azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java]: FYI, <p> is not required if you don't care about having a separate paragraph in the rendered javadoc.

Comment (kunkun-tang) [reply azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java]: Yes, I know that, but I would think we should a separate line for this.

File: azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java (hunk 1 above)
Comment (kunkun-tang) [reply azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java]: Modified as suggested.

**Pull request merged**